tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -431,10 +440,94 @@ def test_common_workload(self):
                                            driver, (workload, validator),
                                            num_workers=self.CLUSTER_NODES - 1,
                                            topology="ensemble")
-        benchmark.start()
-        benchmark_time_min = benchmark.benchmark_time() + 5
-        benchmark.wait(timeout_sec=benchmark_time_min * 60)
-        benchmark.check_succeed()
+        # Latency spikes detection and retry
```

Comment:
This same pattern needs to be applied to test_common_workload and test_retention as those tests are almost identical and both fail with this spike thing.

This could come in the next change which enables this if you want. Perhas the easiest way to do it is to try to combine the code for both tests as much as possible (there is only a small difference in the way retention is configured) which will then lead to both variations naturally getting the spike handling.

Reply:
Would be in the next change along with enabling retry